res/skins/QMLDemo/Deck.qml

```diff
@@ -0,0 +1,263 @@
+import Mixxx 0.1 as Mixxx
+import Mixxx.Controls 0.1 as MixxxControls
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.11
+
+Item {
+    id: root
+
+    required property string group
+
+    ColumnLayout {
+        anchors.fill: parent
+
+        RowLayout {
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+
+            ColumnLayout {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+
+                Rectangle {
+                    id: trackInfo
+
+                    Layout.fillWidth: true
+                    height: 60
+                    color: "#121213"
+
+                    Rectangle {
+                        id: coverArt
+
+                        anchors.top: parent.top
+                        anchors.left: parent.left
+                        anchors.bottom: parent.bottom
+                        width: height
```

Comment:
Not sure if hardcoding the 1:1 aspect ration here is a good idea, but we can keep it like this for the sake of simplicity I guess. I'd prefer if this was documented in an inline comment though.

Reply:
FYI I tried to make a separation between skin-local QML and reusable QML that may be reused by multiple skins, similar to controller mappings vs. midi-components. The latter is in the Mixxx QML module (subdirectory).